bigdata-telemetry/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=INFO,console
```

Comment:
why don't append to file?

Reply:
Well, it should not be here.
I have moved log4j.properties from src/main/... to src/test/...